src/test/resources/testfiles/mock_storage_object_data.json

```diff
@@ -0,0 +1,49 @@
+  51, 34, 44, 34, 107, 105, 110, 100, 34, 58, 34, 115, 116, 111, 114, 97, 103, 101, 35,
+  111, 98, 106, 101, 99, 116, 34, 44, 34, 109, 100, 53, 72, 97, 115, 104, 34, 58, 34, 67,
+  54, 66, 117, 55, 56, 71, 73, 114, 83, 97, 114, 88, 51, 79, 49, 86, 82, 120, 104, 97,
+  119, 92, 117, 48, 48, 51, 100, 92, 117, 48, 48, 51, 100, 34, 44, 34, 109, 101, 100, 105,
+  97, 76, 105, 110, 107, 34, 58, 34, 104, 116, 116, 112, 115, 58, 47, 47, 119, 119, 119,
+  46, 103, 111, 111, 103, 108, 101, 97, 112, 105, 115, 46, 99, 111, 109, 47, 100, 111,
+  119, 110, 108, 111, 97, 100, 47, 115, 116, 111, 114, 97, 103, 101, 47, 118, 49, 47, 98,
+  47, 116, 101, 114, 114, 97, 45, 107, 101, 114, 110, 101, 108, 45, 107, 56, 115, 45, 116,
+  101, 115, 116, 114, 117, 110, 110, 101, 114, 45, 114, 101, 115, 117, 108, 116, 115, 47,
+  111, 47, 49, 49, 50, 50, 101, 102, 49, 100, 45, 98, 53, 49, 99, 45, 52, 48, 102, 49, 45,
+  56, 100, 54, 54, 45, 56, 99, 52, 97, 52, 101, 102, 50, 48, 50, 50, 48, 46, 116, 97, 114,
+  46, 103, 122, 63, 103, 101, 110, 101, 114, 97, 116, 105, 111, 110, 92, 117, 48, 48, 51,
+  100, 49, 54, 50, 54, 53, 51, 56, 54, 49, 56, 48, 55, 50, 48, 50, 51, 92, 117, 48, 48,
+  50, 54, 97, 108, 116, 92, 117, 48, 48, 51, 100, 109, 101, 100, 105, 97, 34, 44, 34, 109,
+  101, 116, 97, 103, 101, 110, 101, 114, 97, 116, 105, 111, 110, 34, 58, 34, 49, 34, 44,
+  34, 110, 97, 109, 101, 34, 58, 34, 49, 49, 50, 50, 101, 102, 49, 100, 45, 98, 53, 49,
+  99, 45, 52, 48, 102, 49, 45, 56, 100, 54, 54, 45, 56, 99, 52, 97, 52, 101, 102, 50, 48,
+  50, 50, 48, 46, 116, 97, 114, 46, 103, 122, 34, 44, 34, 115, 101, 108, 102, 76, 105,
+  110, 107, 34, 58, 34, 104, 116, 116, 112, 115, 58, 47, 47, 119, 119, 119, 46, 103, 111,
+  111, 103, 108, 101, 97, 112, 105, 115, 46, 99, 111, 109, 47, 115, 116, 111, 114, 97,
+  103, 101, 47, 118, 49, 47, 98, 47, 116, 101, 114, 114, 97, 45, 107, 101, 114, 110, 101,
+  108, 45, 107, 56, 115, 45, 116, 101, 115, 116, 114, 117, 110, 110, 101, 114, 45, 114,
+  101, 115, 117, 108, 116, 115, 47, 111, 47, 49, 49, 50, 50, 101, 102, 49, 100, 45, 98,
+  53, 49, 99, 45, 52, 48, 102, 49, 45, 56, 100, 54, 54, 45, 56, 99, 52, 97, 52, 101, 102,
+  50, 48, 50, 50, 48, 46, 116, 97, 114, 46, 103, 122, 34, 44, 34, 115, 105, 122, 101, 34,
+  58, 34, 50, 49, 50, 50, 34, 44, 34, 115, 116, 111, 114, 97, 103, 101, 67, 108, 97, 115,
+  115, 34, 58, 34, 83, 84, 65, 78, 68, 65, 82, 68, 34, 44, 34, 116, 105, 109, 101, 67,
+  114, 101, 97, 116, 101, 100, 34, 58, 34, 50, 48, 50, 49, 45, 48, 55, 45, 49, 55, 84, 49,
+  54, 58, 49, 54, 58, 53, 56, 46, 49, 52, 55, 90, 34, 44, 34, 116, 105, 109, 101, 83, 116,
+  111, 114, 97, 103, 101, 67, 108, 97, 115, 115, 85, 112, 100, 97, 116, 101, 100, 34, 58,
+  34, 50, 48, 50, 49, 45, 48, 55, 45, 49, 55, 84, 49, 54, 58, 49, 54, 58, 53, 56, 46, 49,
+  52, 55, 90, 34, 44, 34, 117, 112, 100, 97, 116, 101, 100, 34, 58, 34, 50, 48, 50, 49,
+  45, 48, 55, 45, 49, 55, 84, 49, 54, 58, 49, 54, 58, 53, 56, 46, 49, 52, 55, 90, 34, 125
+]
```

Comment:
what does this array of bytes represent? If it is a binary file e.g. a .tgz, could we just store the .tgz instead of this array?

Reply:
This is simply the raw `byte[]` array of `mock_event_gzip.json` intended for tests that directly consumes the `byte[]` array. I have added comments to `BaseTest` to describe this file.